src/libraries/Microsoft.Extensions.Primitives/src/Resources/Strings.resx

```diff
@@ -132,4 +132,7 @@
   <data name="Capacity_NotUsedEntirely" xml:space="preserve">
     <value>Entire reserved capacity was not used. Capacity: '{0}', written '{1}'.</value>
   </data>
+  <data name="Error_MissingConfig" xml:space="preserve">
+    <value>'{0}' was set on the provided {1}, but the following properties were not found on the instance of {2}: {3}</value>
```

Comment:
Do you need this both on M.E.Primitives and M.E.ConfigurationBinder?

Reply:
Removed from M.E.Primitives